### ISSUE TITLE
Media & File browsing weirdness

### ISSUE THREAD

**Issue by bdeclerc** [openfed/openfed8#12]
Drag&Drop uploaded a PNG file, went to admin/content to check it out:

admin content page has four tabs
Content --> /admin/content
Files --> /admin/content/files
Files --> /admin/content/files
Media --> /admin/content/media

So "Files" appears twice.

The image is listed under /admin/content/files but not under /admin/content/media.

**Comment by bdeclerc** [comment]
So "Media" is actually the embeddable entities - we may need a better name for this, since this means images aren't considered "Media" but "Files" while really they are both, while an iframe or a javascript isn't really a "Media" item?

Any naming suggestions?

**Comment by bdeclerc** [comment]
"Embeddables"?

**Comment by stefanruijsenaars** [comment]
I like Embeddable 👍

**Comment by rutiolma** [comment]
Just to be sure, you suggest to rename the view "Media" title to "Embeddables"?

About Files, should be a bug somewhere and we'll check.

**Comment by bdeclerc** [comment]
Yeah, not the awesomest of names, but more accurate than "Media" - if someone invents a better name we'll use that :)

**Comment by rutiolma** [comment]
Fix committed. Should be ok for the next release.
